modules/registry-configuring-registry-storage-install-rhodf.adoc

```diff
@@ -0,0 +1,348 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/Configuring-the-registry-for-rhodf.adoc
+
+:_content-type: PROCEDURE
+[id="registry-configuring-registry-storage-install-rhodf_{context}"]
+= Install Red Hat OpenShift Data Foundation to configure storage on the Image Registry Operator for bare metal and vSphere
+
+The Image Registry Operator is not initially available for platforms that do not provide default storage. After installation, you must configure your registry to use storage so that the Registry Operator is made available.
+
+.Prerequisites
+
```

Comment:
@bmcelvee Do we need adding the ODF installing steps?  we can refer the ODF offical docs, right?
Here the oc installed steps are just for qe to test on vsphere, the baremetal installation steps are different, need installed LSO additional. LSO installation need external storage or adding storage volume.
So I suggest to refer ODF offical docs directly.

Reply:
Ah, thanks! I removed them and we will link to the RHODF docs in the additional resources and have the prereqs in place.